src/twisted/conch/ssh/transport.py

```diff
@@ -418,6 +419,11 @@ class SSHTransportBase(protocol.Protocol):
         passed to L{sendPacket} but could not be sent because it is not legal
         to send them while a key exchange is in progress.  When the key
         exchange completes, another attempt is made to send these messages.
+
+    @ivar _supportsExtensionNegotiation: a boolean indicating whether the
+        other side supports RFC 8308 extension negotiation.
```

Comment:
I had to read the docs twice to see what "other" means.
 
I think that `peer` is the generic terminology in twisted to refer to the other side of the connection.

Maybe 

```suggestion
    @ivar _peerSupportsExtensions: a boolean indicating whether the
        other side supports RFC 8308 extension negotiation.
```